starter-code/index.js

```diff
@@ -1,17 +1,66 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos > this.items.length) {
+      throw new Error("OutOfBounds");
+    } else return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if (this.items.length == 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      this.items.sort(function(a, b) {
```

Comment:
I believe the this.items list is sorted every time  you add a new element to it. You will not have to sort it again here